File: 7KYU/Jaden Casing Strings/solution.go
package kata

import "strings"

func ToJadenCase(str string) string {
  
  words := strings.Split(str, " ")
  jCased := []string{}
  
  for _, w := range words {
    jCased = append(jCased, strings.Title(w))  
  }
  
  return strings.Join(jCased, " ")
}

File: 7KYU/Jaden Casing Strings/solution.cs
using System;
using System.Linq;
public static class JadenCase
{
  private static string ToTitleCase(string s)
    => System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(s);
  
  public static string ToJadenCase(this string phrase)
    => string.Join(" ", phrase.Split(" ")
                              .Select(ToTitleCase));
}